Add triangulation guide (#688)
napari/docs: docs/howtos/layers/shapes.md

adjust the position, size, face color, edge color, and opacity of all the shapes
independently, both programmatically and from the GUI.

```{tip}

> To draw Shapes to the screen, napari has to first break them up into triangles.

Reviewer: ```suggestion
To draw Shapes in the viewer, napari first breaks up the shapes into triangles.
```

> It turns out that there's lots of ways to do this, and some are faster than

Reviewer: ```suggestion
It turns out that there are many ways to do this, and some are faster than
```
